### PR TITLE
Adding Cleos Auth & button to login as viewer

### LINKED ISSUE #11: Support "logging in" to view account without authenticating account ownership
With this function, it will be possible to view any blockchain account's token balances and transaction history. (This capability is of course available through block explorers, but the point of Telos Wallet is to be more accessible to general users.) This delivers on the "Blockchain is transparent" claim for a less-tech-savvy population.

A suggested UI is

If the user attempts to submit a _Send_ transaction, it is still possible to present the "Sign manually" pop-up:

FWIW, I find "sign manually" esr actually gives a smoother UX than the UAL login.

@poplexity

### PR DESCRIPTION
**Pull request by Viterbo**
# Fixes #11

## Description
This issue requires a way to explore the web wallet without actual login with a wallet. The best (fast and easy) way to do that is by logging the user in using cleos under the hud. 

## Screenshots
image

## Checklist:

-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have cleaned up the code in the areas my change touches
-   [x] My changes generate no new warnings
-   [x] Any dependent changes have been merged and published in downstream modules
-   [x] I have checked my code and corrected any misspellings
-   [x] I have removed any unnecessary console messages

### CODE REVIEW

File: src/boot/ual.js [hunk 1]
```diff
@@ -15,9 +18,135 @@ export default boot(async ({ app, store }) => {
...
+          Dialog.create({
+            class: 'cleos-auth-dialog',
+            color: 'primary',
+            title: 'Connect to cleos',
+            message: 'Account permission',
+            options: {
+              type: 'radio',
+              model: [],
+              items: [
+                { label: 'Active', value: 'active' },
```
Comment (karynemayer) [inline src/boot/ual.js:63]: I understand that by using `cleos` we need to specify which account authorization we're using, but I wonder if this is an essential step for the end user. 

This issue aims to provide a friendly way for non-technical users to see other people's transactions, so I'm wondering if we could hide this selection from the user when using this feature. 🤔

File: src/layouts/MainLayout.vue [hunk 2]
```diff
@@ -182,7 +182,8 @@ export default {
     },
   },
   async mounted() {
-    await this.autoLogin(this.$route.query.returnUrl);
+    console.log("MainLayout.mounted()");
+    await this.memoryAutoLogin();// this.autoLogin(this.$route.query.returnUrl);
```
Comment (karynemayer) [inline src/layouts/MainLayout.vue:186]: Can we remove the `console.log`?
I would also consider removing the commented code

**Review by karynemayer — CHANGES_REQUESTED** [review]

File: src/boot/ual.js (hunk 1 above)
Comment (donnyquixotic) [reply src/boot/ual.js:63]: "to provide a friendly way for non-technical users to see other people's transactions" if that's the only goal, then there's no point, it's already available via account view in explorer. probably worth revisiting.

File: src/layouts/MainLayout.vue [hunk 3]
```diff
@@ -182,7 +182,8 @@ export default {
     },
   },
   async mounted() {
-    await this.autoLogin(this.$route.query.returnUrl);
+    console.log("MainLayout.mounted()");
```
Comment (donnyquixotic) [inline src/layouts/MainLayout.vue:185]: remove console log

**Review by donnyquixotic — CHANGES_REQUESTED** [review]
request re-discussion

File: src/layouts/MainLayout.vue (hunk 3 above)
Comment (Viterbo) [reply src/layouts/MainLayout.vue]: [removing console log and commented code](https://github.com/telosnetwork/telos-wallet/pull/89/commits/c5ae1c373048f990013ce7cdfde1882dd395fd5b)

File: src/layouts/MainLayout.vue (hunk 2 above)
Comment (Viterbo) [reply src/layouts/MainLayout.vue]: [removing console log and commented code](https://github.com/telosnetwork/telos-wallet/pull/89/commits/c5ae1c373048f990013ce7cdfde1882dd395fd5b)

File: src/boot/ual.js (hunk 1 above)
Comment (Viterbo) [reply src/boot/ual.js:63]: This PR brings two ways of using Cleos authenticator. The normal was adding it to the list of authenticators with a custom couple of handlers (for login and for signing).

The second and the requested way was adding a special button on the home, near the normal login button, which only asks for the name of the account (not for the authorization which is always assumed as 'active'.

Take a look at the title of the dialogs, they read differently depending on the button you used.

"Navigating as viewer" vs "connect to cleos"
image

Comment (Viterbo) [reply src/boot/ual.js:63]: My bad, I still had the last changes locally in my PC and didn't push them. Now I did and you can see what I mean:

[bugfix: actually differentiating the two ways of cleos login](https://github.com/telosnetwork/telos-wallet/pull/89/commits/f9a6f3ca504753695cebb4300d8df4b197f59583)

**Review by karynemayer — APPROVED** [review]

**Review by donnyquixotic — APPROVED** [review]

**Comment by karynemayer** [comment]
I'm going to merged this PR so we follow the process and @Kersten-TCD reviews the entire release instead

**Pull request merged**